packages/editor/src/browser/editor-command.ts

```diff
@@ -100,6 +100,14 @@ export namespace EditorCommands {
         category: 'View',
         label: 'Show All Opened Editors'
     };
+    /**
+     * Command that toggles the minimap.
+     */
+    export const TOGGLE_MINIMAP: Command = {
+        id: 'editor.minimap.enabled',
```

Comment:
We should use a different id, this one is already being used by the preference and it becomes confusing.
https://github.com/theia-ide/theia/blob/d2d4f75897fdb13bcae20409ceef490ea225086e/packages/editor/src/browser/editor-preferences.ts#L129-L132

You should use the same as vscode: https://github.com/microsoft/vscode/blob/7bc29bf67c17608b11e23a8497c8f1de5a0922cf/src/vs/workbench/contrib/codeEditor/browser/toggleMinimap.ts#L15

Reply:
Just changed it.